server/router/placeAndReviews.js

```diff
@@ -0,0 +1,32 @@
+const { getPlace, getPlaceReviews } = require('../models/queries/');
+
+module.exports = (req, res, next) => {
+  const { id, type } = req.query;
+  if (!id || !type) {
+    res.status(400).json({
+      message: 'place id and type required',
+    });
+  }
+  let placeData;
+
+  getPlace(id)
+    .then(places => {
+      if (!places) throw new Error("place doesn't exists");
+      placeData = [places[0].fields, []];
+      return placeData[0];
+    })
+    .then(() => getPlaceReviews(type, id))
+    .then(reviews => {
+      reviews.forEach(record => {
+        placeData[1].push(record.fields);
+      });
+      res.status(200).json({ placeData });
+    })
+    .catch(err => {
+      if (err.message === "place doesn't exists") {
```

Comment:
do not use hardcoded values! You're gonna change the message and forget to change it here. Also you shouldn't use strings for such logic, use error codes, or even better, make your own error types:
```js
function NotFoundError(msg) {
  this.code = 404;
  this.msg = msg;
}

...

throw new NotFoundError(msg);

...

.catch(err => {
  if (err instanceof NotFoundError) {
    res.status(404);
  }
});
```
As suggested in the example, use 404 and not 410.

And lastly, you should have a middleware where you would handle standard errors. So your catch here would look like `.catch(next)` and the error handling would be done in the middleware.